Assets/Scripts/WorldEngine/World.cs

```diff
@@ -1750,24 +1779,9 @@ public void AddFactionToUpdate(Faction faction)
         {
             System.Diagnostics.StackTrace stackTrace = new System.Diagnostics.StackTrace();
 
-            System.Reflection.MethodBase method = stackTrace.GetFrame(1).GetMethod();
-            string callingMethod = method.Name;
-
-            int frame = 2;
-            while (callingMethod.Contains("SetFactionUpdates")
-                || callingMethod.Contains("SetToUpdate"))
-            {
-                method = stackTrace.GetFrame(frame).GetMethod();
-                callingMethod = method.Name;
-
-                frame++;
-            }
-
-            string callingClass = method.DeclaringType.ToString();
-
```

Comment:
Why was this removed?

Reply:
System reflection ops are very CPU and memory intensive. So it's better to remove them if not needed. In most cases it's just simpler to do a ToString() on a stack trace object instead.